docs/connector-development/connector-metadata-file.md

```diff
@@ -129,6 +133,20 @@ releases:
       upgradeDeadline: "2023-12-31"
 ```
 
+#### `forceRelease`
+The optional `scopedImpact` property allows you to bypass check that the `upgradeDeadline` is set to at least one week in the future.
+
+Here's an example:
```

Comment:
Can you add some examples for which `forceRelease` is appropriate versus inappropriate?